src/test/java/com/google/devtools/build/lib/remote/SimpleBlobStoreActionCacheTest.java

```diff
@@ -283,39 +283,6 @@ public void testDownloadDirectoriesWithSameHash() throws Exception {
     assertThat(DIGEST_UTIL.compute(execRoot.getRelative("a/foo/file"))).isEqualTo(fileDigest);
   }
 
-  @Test
```

Comment:
These tests got removed together with the logic from `SimpleBlobStoreActionCache`, but the logic is reused now in `HttpBlobStore`, maybe add these tests for new logic in `HttpBlobStore`?

Reply:
good point. done!